kronolith/lib/Driver/Kolab.php

```diff
@@ -525,9 +525,10 @@ public function delete($calendar)
      */
     protected function _deleteEvent($eventId, $silent = false)
     {
+        $this->synchronize();
+
         if ($eventId instanceof Kronolith_Event) {
             $event = $eventId;
-            $this->synchronize();
```

Comment:
We already call $this->synchronize() in the getEvent method, which is called in the case when eventId is not an instance of Kronolith_Event.